kale/tests/unit_tests/static_analysis/test_ast.py

```diff
@@ -0,0 +1,122 @@
+import ast
+import pytest
+
+from testfixtures import compare
```

Comment:
```___________________________ ERROR collecting kale/tests/unit_tests/static_analysis/test_ast.py ___________________________
ImportError while importing test module '/home/elias/git/hub/arrikto/kale/kale/tests/unit_tests/static_analysis/test_ast.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/unit_tests/static_analysis/test_ast.py:4: in <module>
    from testfixtures import compare
E   ModuleNotFoundError: No module named 'testfixtures'
```
I think testing modules should be installed on `python setup.py install`. That is, `pytest` and `testfixtures`.

Reply:
I think they should be added to `requirements.txt` instead. Tests should be for dev only, if I added them to `setup.py` they would be always `pip`-installed